doc/flame/components.md

```diff
@@ -788,6 +789,18 @@ This is an example of how a quarter-length map looks like:
 
 ![An example of a isometric map with selector](../images/isometric.png)
 
+The component supports negative indexing starting from -2 *(as -1 is reserved for empty tile)* 
```

Comment:
we should consider changing the empty tile to null now that we have NS in Dart. then you can start on -1.
or maybe have another field that defines how to interpret the index (then we could have even more options)
thoughts @spydon @erickzanardo @wolfenrain @renancaraujo

Reply:
One of those options sounds good, with the second one we wouldn't need to have these negative overflow indexes that are used in this PR right?

Reply:
by 2nd option do mean passing a comparison function returning bool to the component in order to determine which indices to use propSize for?